doctrine/annotations/2015-08-31.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Security Misconfiguration Vulnerability in various Doctrine projects
+link:      http://www.doctrine-project.org/2015/08/31/security_misconfiguration_vulnerability_in_various_doctrine_projects.html
+cve:       CVE-2015-5723
+branches:
+    1.0.x:
```

Comment:
Do non-existing branches make any difference? I couldn't find any way to match ranges that are not covered by a branch anymore.